tests/handlers/http/test_to_response.py

```diff
@@ -120,7 +120,7 @@ def test_function() -> Response:
     with create_test_client(test_function) as client:
         http_route: HTTPRoute = client.app.routes[0]  # type: ignore
         route_handler = http_route.route_handlers[0]
-        response = await route_handler.to_response(data=route_handler.fn(), plugins=[], app=None)  # type: ignore
+        response = await route_handler.to_response(data=route_handler.fn(), plugins=[], app=None, request=None)  # type: ignore
```

Comment:
@Goldziher I wanted to keep test_to_response.py simple - hence passing `None` instead of `request` as we need to test that `handler.to_response` returns proper response and request doesn't really matter here.

Reply:
sure but it will save you on a lot of type ignores. just make one request instance nad use it - will actually save you code.

Reply:
It is 4AM for me so I will update the code tomorrow in the morning.

I should note though that `# type: ignore` comments won't go away because of `app=None` and `data=route_handler.fn()` arguments